rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -262,6 +262,7 @@ def submit_questionnaire_response(self, participant_id, questionnaire_id, race_c
                                                             ("streetAddress", street_address),
                                                             ("city", city),
                                                             ("phoneNumber", phone_number),
+                                                            ("loginPhoneNumber", login_phone_number),
```

Comment:
In order for this to populate stuff, you'll need to update the relevant questionnaire JSON to have a "loginPhoneNumber" linkId... so probably here:

https://github.com/all-of-us/raw-data-repository/blob/3bd458de95d02a3a2bf802089453975df0eca016/rest-api/test/test-data/study_consent.json#L78

and here:

https://github.com/all-of-us/raw-data-repository/blob/3bd458de95d02a3a2bf802089453975df0eca016/rest-api/test/test-data/questionnaire3.json#L87

Otherwise the linkId in the questionnaire response won't resolve to anything in the questionnaire, and the answer will be ignored.

Reply:
Hi Dan, this is done, please review.